python/tank/authentication/user_impl.py

```diff
@@ -173,8 +173,8 @@ def __init__(self, host, login, session_token, http_proxy, password=None, cookie
             the session token will be looked for in the users file.
         :param http_proxy: HTTP proxy to use with this host. Defaults to None.
         :param password: Password for the user. Defaults to None.
-        :param cookies: String of raw cookies for the user. Defaults to None.
-        :param saml_expiration: Int time in UTC in second when the SAML claims will expire (in SSO mode)
+        :param cookies: String of raw cookies for the user when using SSO. Defaults to None.
+        :param saml_expiration: Int or None time in UTC in second when the SAML claims will expire, if using SSO.
```

Comment:
How about
`:param int saml_expritation: UTC time in seconds when the SAML claims will expire. Can be None if not using SSO.`
?